src/routes/index.ts

```diff
@@ -1,7 +1,10 @@
 import express from 'express';
+import feedbackReportsRoutes from './feedback-reports/feedback-reports';
+import bugReportsRoutes from './bug-reports/bug-reports';
```

Comment:
You should be putting an `index.ts` in each of those folders so you can just do `import xxxx from './xxxx-reports';`